templates/toss/src/skills/toss.ts

```
@@ -142,12 +135,17 @@
     return;
   }
 
+  const tossDBClient = await getRedisClient();
   if (params.description && params.options && !isNaN(Number(params.amount))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)